app/components/app_session_overview_tallies_component.rb

```diff
@@ -38,22 +38,34 @@ def edit_button_text
     dates.empty? ? "Schedule sessions" : "Edit session"
   end
 
-  def cards_for_programme(programme)
-    stats_for_programme(programme)
-      .except(:eligible_children)
-      .map do |key, value|
-        string_key = key.to_s
-        {
-          heading: card_heading_for(string_key, programme:),
-          colour: card_colour_for(string_key),
-          count: value.to_s,
-          link_to: card_link_to_for(string_key, programme:)
-        }
-      end
+  def card_rows_for_programme(programme)
+    stats =
+      stats_for_programme(programme).except(:eligible_children).stringify_keys
+
+    cards = stats.map { |key, value| card_for(key, value, programme:) }
+
+    if cards.count <= 5
```

Comment:
I think you could replace the `if/else` with `cards.in_groups(cards.count > 5 ? 2 : 1, false)`